.eslintrc.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 6,
+    sourceType: 'module'
+  },
+  extends: [
+    'eslint:recommended',
+    'plugin:ember-suave/recommended'
+  ],
+  env: {
+    browser: true
+  },
+  rules: {
+    'one-var': ['error', 'never']
```

Comment:
I guess that's something that we should discuss next week. Maybe we can create our own ruleset that we'd just reuse across all projects.